beacon_node/lighthouse_network/src/rpc/methods.rs

```diff
@@ -152,13 +164,37 @@ impl<E: EthSpec> MetaData<E> {
                 syncnets: Default::default(),
             }),
             md @ MetaData::V2(_) => md.clone(),
+            MetaData::V3(metadata) => MetaData::V2(MetaDataV2 {
+                seq_number: metadata.seq_number,
+                attnets: metadata.attnets.clone(),
+                syncnets: metadata.syncnets.clone(),
+            }),
+        }
+    }
+
+    /// Returns a V2 MetaData response from self by filling setting 0 custody.
```

Comment:
Wrong doc comment, and inconsistent return of variants, plus code sets to 1 not 0. Where is this function used? Should default to custody requirement